src/_addCorruptFileErrorToReport.ts

```diff
@@ -0,0 +1,33 @@
+import _setWith from "lodash/setWith";
+import { ErrorReport } from "./migration.types";
+
+/**
+ * Reports that no entry matching `fileId` was found in the `/ui/${contentType}/structure` folder.
```

Comment:
```suggestion
 * Adds the error stating `/ui/${contentType}/structure` folder has no entry for `fileId` to `errorReport`
```
I find this phrasing a bit more accurate, and more in line with `_addErrorToReport`'s description